tests/pom.xml

```diff
@@ -223,8 +224,11 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
               <images>
                 <!-- ##### ActiveMQ Broker ##### -->
                 <image>
+                  <name>eclipsehono/hono-test-activemq</name>
                   <alias>activemq</alias>
-                  <name>rmohr/activemq:${activemq.version}</name>
+                  <build>
+                    <from>rmohr/activemq:${activemq.version}</from>
+                  </build>
```

Comment:
why chat on IRC about this. I removed the useless image building but inverting the position of hono server and router/activemq in the pom.xml file. It's a bug in the docker maven plugin (I had a chat with the fabric8 team about that)